src/libraries/System.Data.OleDb/src/DbBindings.cs

```diff
@@ -4,13 +4,16 @@
 
 using System.Data.Common;
 using System.Diagnostics;
+using System.Runtime.InteropServices;
 using System.Text;
 
 namespace System.Data.OleDb
 {
     internal sealed class Bindings
     {
+        private static readonly bool s_runningOnX86 = RuntimeInformation.ProcessArchitecture == Architecture.X86;
```

Comment:
The old defines referenced  "ARCH_arm". (I assume that referred to 32 bit.) Do we expect this library to work on ARM/ARM64? If so will the right things happen?

Reply:
Yes, the right things will happen for ARM/ARM64. This is the condition that we are replicating from `oledb.h` from Windows SDK:
```
#if defined(_WIN64) || defined(_ARM_)
#include <pshpack8.h>	// 8-byte structure packing
#else
#include <pshpack2.h>	// 2-byte structure packing
#endif
```